src/app/pages/Home/Home.tsx

```diff
@@ -10,10 +11,10 @@ export default function Home(): JSX.Element {
       <Header></Header>
       <p className={styles.userInfo}>Your location:</p>
       <LocationCard
-        locationName="München"
+        locationName={useRevGeoCoding()}
```

Comment:
This is a hook an should not be called here. Hooks should only be called at the very top of the react functional component.